CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -260,7 +260,84 @@ The winners of block forks and slot battles are determined by which pool has a l
 
 ## Leverage-based Pool Ranking Recommendation
 
-Pool ranking scores in wallet browsers have a significant impact on delegator choices. When pledge becomes the most important factor for total pool size lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an ascending order with the lowest leverage pools first.
+Pool ranking scores in wallet browsers have a significant and powerful game theory impact on delegator pool choices, yet it is often overlooked and not transparent in wallets.
+
+When pledge becomes the most important factor for total pool size, lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an ascending order with the lowest leverage pools first.
```

Comment:
Ascending order likely needs to be changed to descending order.

Reply:
updated